File: Yet Another Two Integers Problem.py
for _ in range(int(input())):
    a, b = map(int, input().split())
    if a == b:
        print(0)
    elif a > b:
        print(((a - b) // 10) if ((a - b) % 10 == 0) else (((a - b) // 10) + 1))
    else:
        print(((b - a) // 10) if ((b - a) % 10 == 0) else (((b - a) // 10) + 1))

File: Yet Another Two Integers Problem.cpp
#include <iostream>
#include <cstdlib>

using namespace std;

int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    int t, a, b;
    cin >> t;
    while (t--) {
        cin >> a >> b;
        if (a == b) {
            cout << 0 << "\n";
        }
        else {
            if ((abs(a - b) % 10) == 0) {
                cout << abs(a - b) / 10 << "\n";
            }
            else {
                cout << (abs(a - b) / 10) + 1 << "\n";
            }
        }
    }
    return 0;
}

/* Setter's solution
#include <bits/stdc++.h>

using namespace std;

int main() {
#ifdef _DEBUG
	freopen("input.txt", "r", stdin);
//	freopen("output.txt", "w", stdout);
#endif

	int t;
	cin >> t;
	while (t--) {
		int a, b;
		cin >> a >> b;
		cout << (abs(a - b) + 9) / 10 << endl;
	}

	return 0;
}
*/